src/components/Navigation/Navigation.tsx

```diff
@@ -38,37 +51,63 @@ const Navigation = (props: NavigationProps) => {
               className="navigation__buttons-profile-button"
               onClick={() => setIsProfileOpen(!isProfileOpen)}
             >
-              <img className="navigation__buttons-profile-image" src={props.profileImage} alt="" />
+              <img
```

Comment:
can you add null checks on these to make sure the props are there before displaying?